integration-tests/tests/mysql.test.ts

```diff
@@ -1263,3 +1263,12 @@ test.serial('prefixed table', async (t) => {
 
 	await db.execute(sql`drop table ${users}`);
 });
+
+test.serial('timestamp timezone', async (t) => {
+	const { db } = t.context;
+
+	await db.insert(usersTable).values({ name: 'John' });
```

Comment:
Could you add a test inserting a Date with a non-default timezone specified? I'm curious if it'll change the date string format returned from the driver on selection.

Reply:
Good point! I added the test as you specified for both mysql and postgres timestamps. Upon running the tests I found out that timestamps are by default inserted respecting the client's timezone as well, instead of in UTC format. I pushed a fix for this behavior, which passed all the tests.